definitions/variable/Electricitytags_add.yaml

```diff
@@ -0,0 +1,41 @@
+
+- Electricity Source:
+    - Gas|CCGT|w/ CCS:
```

Comment:
in PR #7, @enokandi suggested to define turbines not as abbreviations but as "Combined Cycle", "Gas Turbine", etc.

Please agree on a common approach before we proceed with merging these PRs.